packages/devtools_testing/fixtures/riverpod_app/lib/main.dart

```diff
@@ -0,0 +1,61 @@
+// ignore: unused_import, allows the tests to use functions from tester.dart
```

Comment:
add copyright notices to all dart files matching the ones in the rest of devtools.